.eslintrc.js

```diff
@@ -1,45 +1,132 @@
 module.exports = {
+  'root': true,
   'extends': [
-    'eslint:recommended',
-    'plugin:json/recommended'
+    'plugin:@okta/okta/courage-app' // apply courage-app rules to all files including properties
   ],
-  'env': {
-    'browser': true,
-    'jasmine': true,
-    'node': true,
-    'amd': true
-  },
   'parser': 'babel-eslint',
   'parserOptions': {
     'sourceType': 'module',
     'ecmaVersion': 2017
   },
-  'plugins': [
-    'eslint-plugin-local-rules',
-    'eslint-plugin-no-only-tests'
-  ],
-  'globals': {
-    'spyOnEvent': false,
-    'JSON': true,
-    'DEBUG': true,
-    'Promise': true
-  },
   'overrides': [
     {
-      'files': ['src/**/*.js'],
+      // temporarily ignoring files that violate the @okta/okta/no-exclusive-language rule
+      'files': [
+        'packages/**/login.properties',
+        'packages/**/country_fr.properties',
+        'buildtools/commands/generate-phone-codes.js',
+        'stylelint.config.js',
+        'webpack.playground.config.js'
```

Comment:
For violations in JS files, can we add the ignore comment in the file to avoid block-listing it here? For example:

```
// eslint-disable-next-line @okta/okta-ui/no-exclusive-language
const mainBranch = 'master';
```

Reply:
hah, I was going to do that in a follow up PR 😄 . I'll try and include them in this PR

Reply:
Yeah - would prefer we do it now to avoid another round of review. Should be pretty straightforward! 😄